src/main/java/seedu/financeit/utils/SaveManager.java

```diff
@@ -0,0 +1,133 @@
+            int entsize = EntList.getItemsSize();
+            for (int x = 0; x < entsize; x++){
+                Entry ent = (Entry) EntList.getItemFromIndex(x);
+                str.append(ent.toString() + System.lineSeparator());
+            }
+        }
+        FileWriter fw = new FileWriter("./data/save.txt");
+        fw.write(String.valueOf(str));
+        fw.close();
+    }
+    public static void load() throws IOException{
+        if(!Files.exists(Paths.get("./data"))){
+            Files.createDirectory(Paths.get("./data"));
+        }
+        if(!Files.exists(Paths.get("./data/save.txt"))){
+            Files.createFile(Paths.get("./data/save.txt"));
+        }
+        File f = new File("./data/save.txt");
+        Scanner s = new Scanner(f);
+        InputParser in = new InputParser();
+        String buffer[];
+        int ledger_index = -1;
+        while (s.hasNext()) {
+            String str = s.nextLine();
+
+            if(str.contains(";")){
+                buffer = str.split(";", 5);
+                if (buffer[0].contentEquals("Expense")){
+                    buffer[0] = " -e";
+                } else if (buffer[0].contentEquals("Income")) {
+                    buffer[0] = " -i";
+                }
+                switch (buffer[1]){
+                    case "TRANSPORT": buffer[1] = "tpt";
+                        break;
+                    case "FOOD": buffer[1] = "fd";
+                        break;
+                    case "TRAVEL": buffer[1] = "tvl";
+                        break;
+                    case "SHOPPING": buffer[1] = "shp";
+                        break;
+                    case "BILLS": buffer[1] = "bll";
+                        break;
+                    case "SALARY": buffer[1] = "slr";
+                        break;
+                    case "ALLOWANCE": buffer[1] = "alw";
+                        break;
+                    default:
+                }
+                if (buffer[3].length() >= 4) {
+                    buffer[3] = charRemoveAt(buffer[3], 2);
+                }
+                EntryTracker.setCurrLedger((Ledger)ManualTracker.getLedgerList().getLedgerByIndex(ledger_index));
+                EntryTracker.packet = in.parseInput("entry new /time " + buffer[3] + " /cat " + buffer[1] + " /desc " + buffer[4] + " /amt " + buffer[2] + buffer[0]);
+                EntryTracker.handleCreateEntry();
+
+
+            } else {
+                buffer = str.split(" ", 3);
+                switch(buffer[0]){
+                    case "Jan": buffer[0] = "01";
+                        break;
+                    case "Feb": buffer[0] = "02";
+                        break;
+                    case "Mar": buffer[0] = "03";
+                        break;
+                    case "Apr": buffer[0] = "04";
+                        break;
+                    case "May": buffer[0] = "05";
+                        break;
+                    case "Jun": buffer[0] = "06";
```

Comment:
You can try reviewing LocalDateTime methods for better ways to parse months. Likewise, my DateTimeManager class and parseRawDateTime method in InputParser class may be helpful to you in this particular function.

Reply:
In case you cannot find a better way to deal with this, look at my previous comment